GliaWidgets/Sources/EngagementLauncher/EngagementLauncher.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import GliaCoreSDK
+
+public final class EngagementLauncher {
```

Comment:
This needs documentation as well because it is publicly accessible.

Reply:
Added https://github.com/salemove/ios-sdk-widgets/pull/1052/commits/11a0de0903d86dcb08860ae8475dd1933ac39791